var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -468,11 +471,20 @@ def configure_args(self):
         if spec.satisfies('+binutils'):
             binutils = spec['binutils'].prefix.bin
             options.extend([
-                '--with-sysroot=/',
                 '--with-gnu-ld',
                 '--with-ld=' + binutils.ld,
                 '--with-gnu-as',
                 '--with-as=' + binutils.join('as'),
+            ])
+        if spec.satisfies('+static_stage1'):
+            stage1_ldflags = str(self.rpath_args)
+            boot_ldflags = stage1_ldflags + ' -static-libstdc++ -static-libgcc'
+            if '%gcc' in spec:
+                stage1_ldflags = boot_ldflags
```

Comment:
We explicitly removed these two lines in f0275d7e1b31b27c584c17eaf77d1495c2cd222e to fix the build on RHEL. Did you check whether it still works?

Reply:
No, I was just checking the +binutils part; I split the rest out because it was breaking the build with +binutils. .  I can't figure out in what case that +static_stage1 would work, (somewhere they actually ship the static libs with gcc?)  but I figured *someone* put it in there for a reason, so I split it into a separate flag, 'cause  I needed a gcc with the latest binutils to check if that fixed a code generation problem.

Reply:
I pulled them out.  Sorry 'bout that.  There were a lot of changes in there from where I branched off a few months ago, and I missed it.